git-hooks/pre-push

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env php
+
+<?php
+
+$projectName = basename(getcwd());
+
+exec('vendor/bin/phpunit', $output, $returnCode);
+
+if ($returnCode !== 0) {
+  $minimalTestSummary = array_pop($output);
+  printf("Test suite for %s failed: ", $projectName);
+  printf("( %s ) %s%2\$s", $minimalTestSummary, PHP_EOL);
```

Comment:
What does this line print? 

- I don't know what `%2` does.
- There is no variable `$s`.